e2e/autoscript-apps/src/authn-oauth/autoscript.ts

```diff
@@ -111,6 +116,24 @@ function autoscript() {
           return newTokens;
         },
       ),
+
+      rxDelay(delay),
+      mergeMap(
+        (tokens) => {
+          console.log('Proactively refresh tokens if expiring soon');
+          rxDelay(delay);
```

Comment:
Yes, you're entirely right. I required a 2 second delay to let the "expiring in 1 second" tokens expire in order to exercise the case where the tokens had expired and **getTokens()** is called. In doing so, I got a little confused about the effect **rxDelay** would have in the test and was thinking along the lines of a **setTimeout**. I think the "tokens expired" test case passed despite this. I've now changed the test so that the **rxDelay** is just above the **mergeMap** and I can now see that it works correctly.